app/src/main/java/com/sopt/instagram/domain/entity/FriendStories.kt

```diff
@@ -0,0 +1,8 @@
+package com.sopt.instagram.domain.entity
+
+data class FriendStories(
+    val isSpecial: Boolean,
+    val memberId: Int,
+    val memberImageUrl: String,
+    val memberName: String,
+)
```

Comment:
```suggestion
data class FriendStory(
    val isSpecial: Boolean,
    val memberId: Int,
    val memberImageUrl: String,
    val memberName: String,
)
```
리스트를 포함하지 않는 단일 데이터 클래스이기 때문에 단수형을 쓰는 건 어떨까요?

Reply:
오 넵! api가 friend/stories라 그렇게 지었었는데 FriendStory가 좋은 것 같네요!